src/ui/views/ManualDownload.svelte

```diff
@@ -31,39 +110,81 @@
     />
   </div>
   <div class="col-6">
-    <h4 style="font-weight: bold;">Manual Download</h4>
+    <h4 style="font-weight: bold;">Manual Download Required</h4>
     <p>
-      The <b>{$manualDownloadGroup}</b> file
-      <b>{$manualDownloadFileData.name}</b> needs to be manually downloaded due to
+      The <strong>{$manualDownloadGroup}</strong> file
+      <strong>{$manualDownloadFileData.name}</strong> needs to be manually downloaded due to
       licensing restrictions. Sorry for the inconvenience!
     </p>
-    <p>
-      Please download the file <b>{$manualDownloadFileData.name}</b> from
-      <a href={$manualDownloadFileData.url}>{$manualDownloadFileData.url}</a>.
-    </p>
-    <p>
-      Once you have it, select it in the file picker and click the button below
-      to continue.
-    </p>
-    <div class="input-group" style="margin-bottom: 1em;">
-      <div class="custom-file">
-        <input
-          type="file"
-          bind:files={downloadedFile}
-          disabled={checkingFile}
-        />
+    
+    <div class="alert alert-info">
```

Comment:
Colors overload, please revert the styling to the simple one that was here before - any redesigns would be project-wide, not in one place.